### PR TITLE
Update to Jetpack WindowManager 1.1.0

### PR DESCRIPTION
**Pull request by romanofranz**
Update the WindowManager sample to the latest stable version

### CODE REVIEW

**Comment by romanofranz** [comment]
Fix: #36

**Review by yaraki — APPROVED** [review]

File: samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at
 
-       https://www.apache.org/licenses/LICENSE-2.0
+       http://www.apache.org/licenses/LICENSE-2.0
```
Comment (marcelpinto) [inline samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml:9]: why?

File: samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml [hunk 2]
```diff
@@ -16,22 +16,22 @@
...
+    <!-- Rules for SplitActivityList -->
+
     <SplitPairRule
-        window:splitRatio="0.3"
-        window:splitMinWidth="600dp"
         window:finishPrimaryWithSecondary="always"
         window:finishSecondaryWithPrimary="adjacent">
         <SplitPairFilter
-            window:primaryActivityName="com.example.platform.ui.windowmanager.embedding.SplitActivityList"
+            window:primaryActivityName="com.sa"
```
Comment (marcelpinto) [inline samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml:26]: Is this intended? what's com.sa?

**Review by marcelpinto — CHANGES_REQUESTED** [review]
Could you review your license header template? It seems that it overrided the project one and changed the url to be http instead of https

File: samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml (hunk 1 above)
Comment (romanofranz) [reply samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml:9]: On the browser it redirects to https, but http is the recommended text to add in the header: https://www.apache.org/licenses/LICENSE-2.0#apply

File: samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml (hunk 2 above)
Comment (romanofranz) [reply samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml]: Thanks for catching it! Fixed

**Review by marcelpinto — APPROVED** [review]

**Pull request merged**